build_production/sitemap.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+ <url>
+  <loc>https://jigsaw.tighten.co</loc>
```

Comment:
Interestingly, it's not `https` ... but this URL works somehow? I'd rather we make it be https, so let me see if we can just do that on our side.

Reply:
OK, it's GitHub Pages. Let's make this http for now (so could you change it in config and then re-build and re-commit)? Thanks!

Reply:
Gotta get this over on Netlify ;) I've updated it!

Reply:
haha, yes!